### ISSUE TITLE
Multiple number blank problem

### ISSUE THREAD

**Issue by ConorOBrien-Foxx** [cheddar-lang/Cheddar#14]
**Version:** 0.1.2

### Problem

Blanks in numbers. Multiple blanks should be allowed, such as `3___5`.

### Desired Behavior

`3__5` => `35`, not error.

### How to Reproduce

 1. Open up the repl. Type `3__5`. Voila.

**Comment by vihanb** [comment]
What error are you receiving?

**Comment by ConorOBrien-Foxx** [comment]
```
λ node ./dist/cli/repl.js
cheddar> 5__5
readline.js:945
            throw err;
            ^

TypeError: Cannot read property '0' of undefined
    at CheddarExec.exec (C:\Users\Conor O'Brien\Documents\Programming\Cheddar\Cheddar\dist\interpreter\exec.js:54:29)
    at Interface.<anonymous> (C:\Users\Conor O'Brien\Documents\Programming\Cheddar\Cheddar\dist\cli\repl.js:65:24)
    at emitOne (events.js:96:13)
    at Interface.emit (events.js:188:7)
    at Interface._onLine (readline.js:224:10)
    at Interface._line (readline.js:566:8)
    at Interface._ttyWrite (readline.js:843:14)
    at ReadStream.onkeypress (readline.js:115:10)
    at emitTwo (events.js:106:13)
    at ReadStream.emit (events.js:191:7)
```

**Comment by vihanb** [comment]
Fixed in e2e9ced11fd16d741b5638b4c2a74e20e4bb6fbd
